modules/update-process-workflow.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * updating/understanding_updates/how-updates-work.adoc
+
+:_content-type: CONCEPT
+[id="update-process-workflow_{context}"]
+= Update process workflow
+
+The following represents a detailed workflow of the {product-title} (OCP) update process.
+
+. The target version is stored in the `spec.desiredUpdate.version` field of the ClusterVersion resource, which may be managed through the web console or the CLI.
+
+. The Cluster Version Operator (CVO) discovers that the `desiredUpdate` in the ClusterVersion resource differs from the current cluster version.
```

Comment:
maybe s/discovers/determines 

Anthropomorphism? Not sure why _determines_ seems less anthropomorphic.

Reply:
Would "observes" be better?

Reply:
Sure, "detects" works for me as long as it's not technically inaccurate.